edb/lib/std/30-arrayfuncs.edgeql

```diff
@@ -41,6 +41,41 @@ std::array_unpack(array: array<anytype>) -> SET OF anytype
 };
 
 
+CREATE FUNCTION
+std::array_fill(val: anytype, n: std::int64) -> array<anytype>
```

Comment:
I wonder if we should add `repeat(val: anytype, n) -> set of anytype` instead.  Then this could be expressed as `array_agg(repeat('foo', 10))`

Reply:
I'm tempted to say that it's worth having a `repeat` function all by itself in addition to `array_fill`, largely because calling an `array_agg` on it sounds like it would not be efficient at all, since aggregates in SQL are a whole category of their own that has to walk the data in a linear fashion.